00-Introducing-HOOMD-blue/03-Initializing-the-System-State.ipynb

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">x = numpy.linspace(-L / 2, L / 2, K, endpoint=False)
</pre><p>This is the way I would do it. This is what NumPy recommends to use for non-integer spacings.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>done.</p>